src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -780,9 +759,33 @@ bool layout_optimizer::deconvolution_b_fs_yx_fsv16_opt(layout const &input_layou
     return false;
 }
 
+bool layout_optimizer::needs_all_usr_onednn_bfyx_to_blocked(format fmt_prev, const program_node& node) {
+    bool all_users_match = true;
+    for (auto usr : node.get_users()) {
+        if (!usr->is_type<convolution>()) {
+            all_users_match = false;
+            break;
+        }
+
+        auto input_layout = node.get_output_layout();
+        auto& conv = usr->as<convolution>();
+        auto conv_output_layout = conv.get_output_layout();
+        auto weights_layout = conv.weights(0).get_output_layout();
+        format expected_conv_fmt = get_expected_layout(conv_output_layout, conv, weights_layout).format;
+        if (!needs_onednn_bfyx_to_blocked(fmt_prev, expected_conv_fmt, input_layout, conv)) {
+            all_users_match = false;
+            break;
+        }
+    }
+
+    return all_users_match;
+}
+
 bool layout_optimizer::needs_onednn_bfyx_to_blocked(format fmt_prev, format fmt_next, layout& prev_output_layout, const convolution_node& node) {
     auto next_output_layout = node.get_output_layout();
-    if (prev_output_layout.data_type != next_output_layout.data_type)
+    if (!(prev_output_layout.data_type == next_output_layout.data_type ||
+        (prev_output_layout.data_type == data_types::i8 && next_output_layout.data_type == data_types::u8) ||
+        (prev_output_layout.data_type == data_types::u8 && next_output_layout.data_type == data_types::i8)))
```

Comment:
This condition is difficult to understand.
Question: is `i8 -> u8` not supported in onednn?

Reply:
There is !